README.md

```diff
@@ -4,6 +4,9 @@ Codebase for Sparkle, brought to you by Sparkle.
 
 ## Getting started
 
+You can get your first experience with Sparkle at staging: \
+https://staging.sparkle.space/in/devaliashacksville
+
```

Comment:
Eventually it would be good to add a 'setting up a new firebase environment' section here, though that is likely OOS of this PR.

We have https://github.com/sparkletown/internal-sparkle-docs/pull/5 to add this to our internal documentation, which I expect/plan to eventually extract from there and copy over to or `docs/` section in this repo, then link it from this README.

Perhaps we could add a section + short note here saying if you're interested in setting up the project to run in your own environment, to open an issue asking for assistance, or something like that? That way we can capture some 'signal' about people who might be wanting to try, but 'turn away' due to not having the full instructions available yet.

---

There are also some new scripts I wrote/added in https://github.com/sparkletown/sparkle/pull/1298 to simplify bootstrapping a new environment once it's firebase project is set up/configured.

Off the top of my head, the following scripts are likely quite useful in setting up a new environment (after the manual steps of creating/configuring the firebase project)

- https://github.com/sparkletown/sparkle/blob/staging/scripts/bootstrap-new-environment.ts
- https://github.com/sparkletown/sparkle/blob/staging/scripts/create-users.ts
- https://github.com/sparkletown/sparkle/blob/staging/scripts/update-admin-role-users.ts

Eventually we will probably capture the manual firebase setup steps in a script or similar as well, but for now, this remains manual.